9jaswag/PostIt
# #148287761 Maintain defined standards

#### Description of Task to be completed?
- refactors the ```findUser``` controller to return just one user
- displays prompts when adding/removing a user from a group
- refactors the user search to return users on keystroke

#### How should this be manually tested?
- visit the app, try adding/removing a user, you should see a prompt
- visit the app, try searching for a user, results should appear as you type

#### Any background context you want to provide?
- None

#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/148287761

#### Screenshots (if appropriate)
![screen shot 2017-10-24 at 12 58 13 pm](https://user-images.githubusercontent.com/8125356/31941809-0c0ac2d8-b8bb-11e7-920a-a73751e96554.png)


#### Questions:
- None
